comm/communicator.go

```
@@ -75,6 +75,50 @@ func NewCommunicator(in io.ReadCloser, db db.DB) *Communicator {
 
 		},
 	})
+	comm.shell.AddCmd(&ishell.Cmd{
+		Name: "prefix",
+		Help: "prefix <bucket> <prefix>: show list of keys with given prefix",
+		Completer: func(args []string) []string {
+			if len(args) == 0 {
+				return comm.db.ListBuckets()
+			}
+			if len(args) == 1 {
+				return comm.db.PrefixScan(args[0], args[1], 0, 100)
```

panics on `prefix bucket <TAB>`

Also we can use proper parsing `nutsCmd, _ := comm.reader.Read("prefix", args)` instead of `args[N]`, ignoring err. Not nesessarily better, but more readable I think

---

FIxed panic.
Calling a function that could return an error and ignoring the error is worse in my book than using elements of a slice.